input/vlct/orszag-tang/orszag-tang.in

```diff
@@ -0,0 +1,76 @@
+# Problem: Orszag-Tang vortex test problem for VLCT
+# Author: Matthew Abruzzo (matthewabruzzo@gmail.com)
+
+# we adopt the initial conditions detailed in the main athena paper:
+# Stone et al. (2008). They are further described at
+
+   include "input/vlct/vlct.incl"
+
+   Domain {
+      lower = [0.0, 0.0, 0.0];
+      upper = [1.0, 1.0, 0.167];
+   }
+
+   Mesh {
+      root_rank = 3; # 3D
+      root_blocks = [2,2,2];
+      root_size = [192,192,6]; # number of cells per axis
+   }
```

Comment:
Are the individual cells not cubic on purpose?

Reply:
Stupid question: could you clarify what you mean about cubic? Are you talking about individual cells being cubes?

Reply:
Yes, I meant cells being cubes in terms of physical dimensions, which is used in every other test (IIRC) but here (1/192 != 0.167/6).

Reply:
No, there's no good reason for this. I can update this if you would like. I'm pretty sure this was arbitrary since it's a 2d problem.

I don't remember exactly what I did here. I imagine I was trying to make z-axis have a length of 1 or something and I wasn't thinking about it right.